### ISSUE TITLE
change wind barb icons?

### ISSUE THREAD

**Issue by docgregt** [Reading-eScience-Centre/ncwms#42]
Hi,
Is there a way to switch the icons for the wind barbs?
I would like to use my own icons.
Thanks,
Greg

**Comment by guygriffiths** [comment]
No, wind barb icons are hardcoded as Path2D objects - there isn't a straightforward way of switching them

**Comment by docgregt** [comment]
Ok. Does your sld support mapping icons to points?

**Comment by guygriffiths** [comment]
Apologies for the delayed response, I was on leave.  The SLD doesn't support arbitrary icons or anything like that, it is just a wrapper for specifying parameters for the various styles we have implemented

**Comment by docgregt** [comment]
Hi, No problem. Thanks for answering! The wind barb rendering seems to be a bit low resolution and I was just looking for alternative to improve the look. I am also having a hard time with the spacing and size. If I increase the size then the spacing gets too far apart and too few icons are drawn.
